packages/app/cypress/e2e/runner/support/verify-helpers.ts

```diff
@@ -0,0 +1,205 @@
+import _ from 'lodash'
```

Comment:
The majority of this file was pre-existing in the runner package [here](https://github.com/cypress-io/cypress/blob/10.0-release/packages/runner/cypress/support/verify-failures.js). It's been modified here to no longer use `runIsolatedCypress`, as the spec is responsible for scaffolding and initializing the e2d test. We also no longer spy on the socket for external IDE interactions and intercept mutations/assert DOM instead.

This might change a bit as more tests get ported over.